budgetkey_data_pipelines/pipelines/tenders/pipeline-spec.yaml

```diff
@@ -0,0 +1,15 @@
+exemptions-publishers:
+  pipeline:
+    - run: add_metadata
+      parameters:
+        name: "tenders-exemptions-publishers"
+    - run: add_resource
+      parameters:
+        name: "mr-gov-il-search-exemption-messages"
+        url: "http://www.mr.gov.il/ExemptionMessage/Pages/SearchExemptionMessages.aspx"
+        format: "txt"
+    - run: stream_remote_resources
+    - run: add_exemptions_publishers_resource
+    - run: dump.to_path
+      parameters:
+        out-path: /var/datapackages/tenders-exemptions-publishers/
```

Comment:
And respectively, the pipeline file should be at:
`budgetkey_data_pipelines/pipelines/procurement/tenders/exemptions`
And the pipeline id should be `publishers`